tests/functional/Tribe/Events/Navigation_Test.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Tribe\Events;
+
+class Navigation_Test extends \Tribe__Events__WP_UnitTestCase {
+	public function setUp() {
+		// before
+		parent::setUp();
+		// your set up methods here
+	}
+
+	public function tearDown() {
+		// your tear down methods here
+		// then
+		parent::tearDown();
+	}
+
+	public function test_closest_event() {
```

Comment:
Looks like we're setting up a linear set of events for this test - should we additionally simulate a pre-existing event having it's datetime amended (re my above comment)?